http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala

```diff
@@ -333,7 +333,7 @@ private[leonardo] object BuildHelmChartValues {
           raw"""ingress.path.sas=${ingressPath}${"(/|$)(.*)"}""",
           raw"""ingress.path.welder=${welderIngressPath}${"(/|$)(.*)"}""",
           raw"""ingress.proxyPath=${ingressPath}""",
-          raw"""ingress.referer=${config.leoUrlBase}""",
+          raw"""ingress.referer=${config.proxyConfig.proxyUrlBase.dropRight(6)}""",
```

Comment:
Would the order of that config list ever change? If so is there a more robust way to grab the element you need?

Reply:
what `dropRight(6)` does is just dropping the right most 6 digits of the String. So instead of `https://https://notebooks.firecloud.org/proxy/`, it'll become `https://notebooks.firecloud.org/`...

Things will break if we ever change how we do proxy path, which I think isn't very likely. maybe I'll add a comment in the config in a follow up

Reply:
Yeah, a comment is perfectly fine I think :) Thanks!

Reply:
It seems like what you need is the URI's Host value, without the `/proxy` part. Could this be changed to `URI(config.proxyConfig.proxyUrlBase).getHost` for clarity? I'm concerned that the `dropRight(6)` is obscuring the more specific ask of "remove trailing `/proxy` from the base URL"

Reply:
oh cool..didn't realize the helper exists..